src/components/GFLMap.tsx

```diff
@@ -53,15 +54,12 @@ export const GFLMap: React.FC<MapProps> = ({
               hospitalInfo: hospital,
             })
           }
-        >
-          <RedPin />
-        </Marker>
+        />
       ))}
       {popupInfo && (
-        <HospitalPopup
-          popupInfo={popupInfo}
-          onClose={() => setPopupInfo(null)}
-        />
+        <div style={{ display: "flex" }}>
```

Comment:
Consider using Box instead of div. There are convenience properties that we may find useful while maintaining the applications

Reply:
I modified it like this as your feedback.


```
      {popupInfo && (
        <Box sx={{ display: "flex" }}>
          <GFLPopup popupInfo={popupInfo} onClose={() => setPopupInfo(null)} />
        </Box>
      )}
```